packages/core/tests/completeLocalModuleRegistrations.test.ts

```diff
@@ -59,7 +59,7 @@ test("when called for the first time but the registration status is already \"re
 
     expect(registry.registrationStatus).toBe("ready");
 
-    await registry.completeModuleRegistrations(runtime);
+    await registry.completeModuleRegistrations(runtime, {});
```

Comment:
i don't think i've ever seen an API that forces me to pass an empty object. This method should most likely have a default value or empty?

Any reason why this synthax is forced?